pyproject.toml

```
@@ -27,8 +27,8 @@ classifiers = [
   "Typing :: Typed",
 ]
 dependencies = [
-  "cryptography>=3.2,<40",
-  "django>=4.2,<5.1",
+  "cryptography>=3.2,<=43",
+  "django>=4.2,<=5.1",
```

Versiones como 5.1.1 también son aceptables (5.1.1 debería ser compatible con 5.1.0).

```suggestion
  "django>=4.2,<5.2",
```